react/components/ProductQuantitySelector/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import { NumericStepper } from 'vtex.styleguide'
+import { FormattedMessage } from 'react-intl'
+
+import styles from './quantitySelector.css'
+
+const ProductQuantitySelector = ({ selectedQuantity, onChange, availableQuantity, maximumAvailableQuantity }) => {
+  const showAvailable = availableQuantity <= maximumAvailableQuantity
+  return (
+    <div className={`${styles.quantitySelectorContainer} flex flex-column mb4`}>
+      <div className="mb3 c-muted-2 t-body">
+        <FormattedMessage id="product-details.quantity" />
+      </div>
+      <NumericStepper
+        size="small"
+        value={selectedQuantity}
+        minValue={1}
+        maxValue={availableQuantity ? availableQuantity : undefined}
```

Comment:
Just 
```suggestion
        maxValue={availableQuantity}
```
Would not have the same effect ? 🤔

Reply:
tenho medo se for 0...

Reply:
Ah beleza